data/json/effects.json

```diff
@@ -896,7 +895,9 @@
         "desc": ["Your %s is starting to feel the damage cold has wrought."],
         "part_descs": true,
         "base_mods": {
-            "pain_max_val": [40],
+		    "hurt_min": [1],
+            "hurt_chance": [12],
```

Comment:
all of the damage of frostbite has already been done. lingering damage might be possible, but in no way is the bulk of frostbite damage suddenly appearing once a person gets inside. pain yes. crippling damage no. this should be at least lower than the damage of being frostbitten.